src/net/telemetry.rs

```diff
@@ -58,6 +58,23 @@ pub struct Telemetry {
 
     /// The CPU temperature in degrees Celsius.
     pub cpu_temp: f32,
+
+    /// Measurements related to Pounder
+    pub pounder: Option<PounderTelemetry>,
+}
+
+/// The structure that holds the telemetry related to Pounder.
+///
+/// # Note
+/// This structure should be generated on-demand by the buffer when required to minimize conversion
+/// overhead.
+#[derive(Copy, Clone, Serialize)]
+pub struct PounderTelemetry {
+    /// The Pounder temperature in degree Celsius
+    pub temperature: f32,
+
+    /// The detected RF power into IN channels
+    pub input_powers: [f32; 2],
```

Comment:
```suggestion
    pub input_power: [f32; 2],
```

It's already plural because it's an array, naming it `input_powers` implies each channel has multiple power measurements